Author: Daniel Baumann <daniel.baumann@progress-technologies.net>
Description: Removing dubious fstab entries in fedora template, thanks to Michael Biebl <biebl@debian.org> (see #633053).

diff -Naurp lxc.orig/templates/lxc-fedora.in lxc/templates/lxc-fedora.in
--- lxc.orig/templates/lxc-fedora.in	2011-07-15 21:05:17.289046939 +0200
+++ lxc/templates/lxc-fedora.in	2011-07-15 21:08:09.057082291 +0200
@@ -86,8 +86,7 @@ EOF
 # /lib/init/fstab: lxc system fstab
 none            /spu                      spufs           gid=spu,optional                  0 0
 none            /tmp                      none            defaults                          0 0
-none            /var/lock                 tmpfs           nodev,noexec,nosuid,showthrough   0 0
-none            /lib/init/rw              tmpfs           mode=0755,nosuid,optional         0 0
+none            /var/lock                 tmpfs           nodev,noexec,nosuid               0 0
 EOF
 
     # reconfigure some services
